fizz.go

```diff
@@ -130,48 +130,80 @@ func (g *RouterGroup) GinRouterGroup() *gin.RouterGroup {
+}
+
 func (g *RouterGroup) PATCH(path string, infos []OperationOption, handlers ...gin.HandlerFunc) *RouterGroup {
-	return g.Handle(path, "PATCH", infos, handlers...)
+	return g.PATCHWithExamples(path, infos, nil, nil, handlers...)
 }
 
 // DELETE is a shortcut to register a new handler with the DELETE method.
+func (g *RouterGroup) DELETEWithExamples(path string, infos []OperationOption, example interface{}, examples map[string]interface{}, handlers ...gin.HandlerFunc) *RouterGroup {
+	return g.HandleWithExamples(path, "DELETE", infos, example, examples, handlers...)
+}
+
 func (g *RouterGroup) DELETE(path string, infos []OperationOption, handlers ...gin.HandlerFunc) *RouterGroup {
-	return g.Handle(path, "DELETE", infos, handlers...)
+	return g.DELETEWithExamples(path, infos, nil, nil, handlers...)
 }
 
 // OPTIONS is a shortcut to register a new handler with the OPTIONS method.
+func (g *RouterGroup) OPTIONSWithExamples(path string, infos []OperationOption, example interface{}, examples map[string]interface{}, handlers ...gin.HandlerFunc) *RouterGroup {
+	return g.HandleWithExamples(path, "OPTIONS", infos, example, examples, handlers...)
+}
+
 func (g *RouterGroup) OPTIONS(path string, infos []OperationOption, handlers ...gin.HandlerFunc) *RouterGroup {
-	return g.Handle(path, "OPTIONS", infos, handlers...)
+	return g.OPTIONSWithExamples(path, infos, nil, nil, handlers...)
 }
 
 // HEAD is a shortcut to register a new handler with the HEAD method.
+func (g *RouterGroup) HEADWithExamples(path string, infos []OperationOption, example interface{}, examples map[string]interface{}, handlers ...gin.HandlerFunc) *RouterGroup {
+	return g.HandleWithExamples(path, "HEAD", infos, example, examples, handlers...)
+}
+
 func (g *RouterGroup) HEAD(path string, infos []OperationOption, handlers ...gin.HandlerFunc) *RouterGroup {
-	return g.Handle(path, "HEAD", infos, handlers...)
+	return g.HEADWithExamples(path, infos, nil, nil, handlers...)
 }
 
 // TRACE is a shortcut to register a new handler with the TRACE method.
+func (g *RouterGroup) TRACEWithExamples(path string, infos []OperationOption, example interface{}, examples map[string]interface{}, handlers ...gin.HandlerFunc) *RouterGroup {
+	return g.HandleWithExamples(path, "TRACE", infos, example, examples, handlers...)
+}
+
 func (g *RouterGroup) TRACE(path string, infos []OperationOption, handlers ...gin.HandlerFunc) *RouterGroup {
-	return g.Handle(path, "TRACE", infos, handlers...)
+	return g.TRACEWithExamples(path, infos, nil, nil, handlers...)
 }
 
 // Handle registers a new request handler that is wrapped
 // with Tonic and documented in the OpenAPI specification.
-func (g *RouterGroup) Handle(path, method string, infos []OperationOption, handlers ...gin.HandlerFunc) *RouterGroup {
+func (g *RouterGroup) HandleWithExamples(path, method string, infos []OperationOption, example interface{}, examples map[string]interface{}, handlers ...gin.HandlerFunc) *RouterGroup {
```

Comment:
Wrong comment, should be on `Handle`, and `HandleWithExamples` should have its own.